client/arg.cpp

```diff
@@ -159,7 +159,18 @@ bool analyse_argv(const char * const *argv, CompileJob &job, bool icerun, list<s
                 /* as above but with extra argument */
             } else if (!strcmp(a, "-MT") || !strcmp(a, "-MQ")) {
                 args.append(a, Arg_Local);
-                args.append(argv[++i], Arg_Local);
+                ++i;
+                if (argv[i][0] == '"') {
```

Comment:
I'm wondering if this is best: it seems like several options could have this problem (space in an argument), is there a better way to handle arguments with spaces?